### ISSUE TITLE
tags and exclusion rules

### ISSUE THREAD

**Issue by theMiddleBlue** [SpiderLabs/owasp-modsecurity-crs#1446]
As @zimmerle explained [here](https://github.com/SpiderLabs/ModSecurity/issues/2109#issuecomment-498648854) the tag matching in v3 is no more regex based.

> [...] the match for a tag in v2 is a regular expression by default, that is why it matches partially a string. When implemented on v3 we made it a string comparison; due to performance reasons [...]

In the link above, you can see why, in v3, tags like `OWASP_CRS/FOO/BAR` are no more useful and why we should split it into three different tags (something like `tag:'OWASP_CRS',tag:'FOO',tag:'BAR'`). I know that it means touching all rules with all problems related to it,  but I think that at some stage in the future, we have to face this problem and maybe review all rules' tag.

Of course, we can solve by replacing all `ctl:ruleRemoveTargetByTag=CRS` with `ctl:ruleRemoveTargetById=910000-999999` as done in #1445 (we're waiting this fix https://github.com/SpiderLabs/ModSecurity/issues/2110 🤣) but things like that are no more true for all v3 users:
https://github.com/SpiderLabs/owasp-modsecurity-crs/blob/c6249fa083bf2b3724f20786a2c2c8790cc2e496/rules/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf.example#L129-L144

should I remove this part from rules/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf.example? If yes, can I do it on #1445 PR?

**Comment by csanders-git** [comment]
Yeah, this should be done -- we need to slightly update the wording -- see the PR :)

**Comment by dune73** [comment]
PR #1445 is meant to fix this in order to bring back libModSecurity v3 compatibility. However, PR #1445 is blocked by another libModSecurity bug (https://github.com/SpiderLabs/ModSecurity/pull/2114). We are thus stuck between a hammer and a hard place.

**Comment by lifeforms** [comment]
This is fixed in v3.2.0. We worked around the problem in ModSecurity 3 by adding the explicit tag to all the rules.
